tests/unit/remote/ssh/test_ssh.py

```diff
@@ -56,19 +45,145 @@ def test_no_path(self):
         remote = RemoteSSH(None, config)
         self.assertEqual(remote.prefix, "/")
 
-    def test_port(self):
-        url = "ssh://127.0.0.1/path/to/dir"
-        config = {"url": url}
-        remote = RemoteSSH(None, config)
-        self.assertEqual(remote.port, remote.DEFAULT_PORT)
-
-        port = 1234
-        url = "ssh://127.0.0.1:{}/path/to/dir".format(port)
-        config = {"url": url}
-        remote = RemoteSSH(None, config)
-        self.assertEqual(remote.port, port)
 
-        port = 4321
-        config["port"] = port
-        remote = RemoteSSH(None, config)
-        self.assertEqual(remote.port, port)
+mock_ssh_config = """
+Host example.com
+   User ubuntu
+   HostName 1.2.3.4
+   Port 1234
+   IdentityFile ~/.ssh/not_default.key
+"""
+
+# compat version for mock library version 2
+# mock 2.0.0 can't iterate in mock_open
```

Comment:
You could just adjust mock version requirements in `tests/requirement.txt` up to the needed version, we just didn't update it for a long time :slightly_smiling_face:

Reply:
Done. I misinterpreted mock documentation "Version 2.0.0 is the last version compatible with Python 2.6." But we have min version 2.7 :)